formspree/scss/dashboard.scss

```diff
@@ -111,6 +111,14 @@
 }
```

Comment:
Are we sure we want to move all these files out of the /static/ folder and put them at the root? I agree it's a bit confusing but iirc there may be tools that package our static content up for CDN delivery, and having it all under /static/ might make that easier? It's been a while...

Reply:
I'm just moving source files. All SCSS code will be on `formspree/scss` and all pretranspiled JS will be on `formspree/js`, but the build system will create a single `/formspree/static/main.css` and a `formspree/static/bundle.js`, so these will be served as always.